### DISCUSSION TITLE
How to connect on AWS RDS and access the Laravel artisan

### DISCUSSION THREAD

**Discussion by kodedewe** [brefphp/bref#1100] [General]
I have already updated the .env database parameter based on RDS information (host, port, etc) but still got 500 error.

Also for the console command, I have follow the bref documentation but error 404.

My serverless yml:
`functions:
    # This function runs the Laravel website/API
    web:
        handler: public/index.php
        timeout: 28 # in seconds (API Gateway has a timeout of 29 seconds)
        layers:
            - ${bref:layer.php-80-fpm}
        events:
            - httpApi: '*'
    # This function lets us run artisan commands in Lambda
    artisan:
        handler: artisan
        timeout: 120 # in seconds
        layers:
            - ${bref:layer.php-80} # PHP
            - ${bref:layer.console} # The "console" layer` 
           
![error-404](https://user-images.githubusercontent.com/82808018/142822200-c37b7b5a-53d8-4eb0-913e-e08f75148e43.png)

**Comment by t-richard** [comment]
The command is missing the region.

Try running this instead

```bash
vendor/bin/bref cli livelync-bref-dev-artisan --region=<your AWS region> -- migrate
```

**Reply by kodedewe** [reply]
Thanks it worked
